app/src/pages/inside/testCaseLibraryPage/allTestCasesPage/allTestCasesPage.tsx

```diff
@@ -124,14 +139,56 @@ export const AllTestCasesPage = ({
 
   const onClearSelection = () => setSelectedRowIds([]);
 
+  const setTestCasesPage = (page: number): void => {
+    const offset = (page - 1) * testCasesPageData.size;
+
+    if (folderId) {
+      dispatch(
+        getTestCaseByFolderIdAction({
+          folderId: Number(folderId),
+          limit: pageSize,
```

Comment:
Let's move this shared props in the variable in this function and use in `getTestCaseByFolderIdAction` and `getAllTestCasesAction`

Reply:
Fixed